server/WebAPI/appsettings.json

```diff
@@ -20,6 +20,11 @@
     "Password": "mail_password"
   },
   "Front": {
-    "AddressFront": "http://localhost:8080"
-  }
+    "AddressFront": "http://me.test.com/"
+  },
+  "GoogleOauth": {
+    "ClientId": "333247262677-ri736kk9bh4c129e44j7i6d18idt4ptu.apps.googleusercontent.com",
+    "ClientSecret": "bY2KG6KJXWJD_ShCO2xUj9am",
```

Comment:
серъезная уязвимость публиковать секреты в исходном коде на гитхабе. так стараются не делать

Reply:
Хорошего места для секретов я пока не знаю (
Я подумал, пока в разработке, это не так важно...

Куда стоит положить в нашем случае ?

Reply:
in our case, you should fill in some General data, for example, if you need to keep a file with a mail password on the server, then we fill in this field as 'password' and store it among ourselves. in your case, you can fill in something similar to "secret", and store the true value in another place, for example, in a text file on your computer)))))
in a word, this should not be on github, because anyone can use it